tests/cli_tests/test_cli.py

```diff
@@ -7,5 +7,5 @@ def test_main():
     runner = CliRunner()
     result = runner.invoke(cli)
 
-    assert result.exit_code == 0
+    assert result.exit_code == 2
```

Comment:
2 seems like an error code. Is this expected?

Reply:
https://github.com/pallets/click/pull/1489